ui/raidboss/data/06-ew/raid/p4s.ts

```diff
@@ -0,0 +1,704 @@
+        },
+      },
+    },
+    {
+      id: 'P4S Periaktoi',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: ['69F5', '69F6', '69F7', '69F8'], source: 'Hesperos' }),
+      netRegexDe: NetRegexes.startsUsing({ id: ['69F5', '69F6', '69F7', '69F8'], source: 'Hesperos' }),
+      netRegexFr: NetRegexes.startsUsing({ id: ['69F5', '69F6', '69F7', '69F8'], source: 'Hespéros' }),
+      netRegexJa: NetRegexes.startsUsing({ id: ['69F5', '69F6', '69F7', '69F8'], source: 'ヘスペロス' }),
+      alertText: (_data, matches, output) => {
+        const pinax: { [id: string]: string } = {
+          '69F5': 'Acid',
+          '69F6': 'Lava',
+          '69F7': 'Well',
+          '69F8': 'Levinstrike',
+        };
+        return output.text!({ pinax: pinax[matches.id] });
+      },
+      outputStrings: {
+        text: {
+          en: 'Be on ${pinax} tile',
+        },
+      },
+    },
+    {
+      id: 'P4S Belone Coils',
+      // This is either DPS towers or tank/healer towers
+      // The roles maked by tower grab tethers
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '69DD', source: 'Hesperos', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '69DD', source: 'Hesperos', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '69DD', source: 'Hespéros', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '69DD', source: 'ヘスペロス', capture: false }),
+      delaySeconds: 2,
+      infoText: (data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Get other role Tower',
+          de: 'Geh in einen Turm einer anderen Rolle',
+          fr: 'Prenez la tour d\'un autre rôle',
+          cn: '踩其他职能的塔',
+          ko: '내 직업군이 아닌쪽 장판 밟기',
+        },
+      },
+    },
+    {
+      id: 'P4S Searing Stream',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6A2D', source: 'Hesperos', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '6A2D', source: 'Hesperos', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '6A2D', source: 'Hespéros', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '6A2D', source: 'ヘスペロス', capture: false }),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'P4S Nearsight',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6A26', source: 'Hesperos', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '6A26', source: 'Hesperos', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '6A26', source: 'Hespéros', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '6A26', source: 'ヘスペロス', capture: false }),
+      alertText: (data, _matches, output) => data.role === 'tank' ? output.tankbustersIn!() : output.getOut!(),
+      outputStrings: {
+        tankbustersIn: {
+          en: 'Get in, tankbuster',
+        },
+        getOut: Outputs.out,
```

Comment:
This should say "Out (avoid tankbusters)" I think.  It's not just get out, and everybody can mitigate this.